%YAML:1.0

loop_skip_interval: 3
loop_history_search_num: 20
loop_distance_threshold: 50.0
loop_temporal_consistency_threshold: 20 # floam
loop_global_registration_threshold: 2.0 # currently too large
loop_local_registration_threshold: 2000 # icp normalized cost

visualize_image: 1
load_previous_pose_graph: 0
loop_save_pcd: 1

# scan context
lidar_height: 2.0
pc_num_ring: 20
pc_num_sector: 60
pc_max_radius: 80.0
num_exclude_recent: 50 # not find candidate within previous num_exclude_recent frames
num_candidates_from_tree: 50 # 10 or 50
search_ratio: 0.1
sc_dist_thres: 0.5
tree_making_period: 10

# registration
normal_radius: 1
fpfh_radius: 1.5
div_factor: 1.4 # Division factor used for graduated non-convexity
use_absolute_scale: 1 # Measure distance in absolute scale (1) or in scale relative to the diameter of the model (0)
max_corr_dist: 0.025 # Maximum correspondence distance (also see comment of USE_ABSOLUTE_SCALE)
iteration_number: 64 # Maximum number of iteration
tuple_scale: 0.95 # Similarity measure used for tuples of feature points.
tuple_max_cnt: 1000 # Maximum tuple numbers.








